### PR TITLE
Init evo-delivery-app

### PR DESCRIPTION
**Pull request by tamirho**
fix: #29

### CODE REVIEW

File: evo-delivery-app/evo-delivery-backend/tsconfig.json [hunk 1]
```diff
@@ -0,0 +1,16 @@
+{
+  "compilerOptions": {
+    "target": "es2016",
+    "lib": ["es6"],
```
Comment (yuval-ya) [inline evo-delivery-app/evo-delivery-backend/tsconfig.json:4]: Are we going to use es6 with react too?

**Review by yuval-ya — COMMENTED** [review]
we probably use other APIs for 'routes' service (u probably know)
the structure is good

File: evo-delivery-app/evo-delivery-backend/src/controllers/driver.controller.ts [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  req: Request,
+  res: Response,
+  next: NextFunction
+) => {
+  const page = req.params.page || 1;
+  const limit = req.params.limit || 10;
+
+  try {
+    const drivers = await driverService.getDrivers({}, page, limit);
+    return res.status(200).json({
```
Comment (yuval-ya) [inline evo-delivery-app/evo-delivery-backend/src/controllers/driver.controller.ts:15]: To be consistence with all services, lets create a JsonResponse object that will have the below params and return it

Comment (yuval-ya) [reply evo-delivery-app/evo-delivery-backend/src/controllers/driver.controller.ts:15]: Just an idea for now

Comment (tamirho) [reply evo-delivery-app/evo-delivery-backend/src/controllers/driver.controller.ts:15]: Nice 👍🏽

**Comment by tamirho** [comment]
> we probably use other APIs for 'routes' service (u probably know)
> 
> the structure is good
> 
> 

Right, Whoever performs the task will have to rewrite it

File: evo-delivery-app/evo-delivery-backend/tsconfig.json (hunk 1 above)
Comment (tamirho) [reply evo-delivery-app/evo-delivery-backend/tsconfig.json:4]: Yep

**Pull request merged**